### PR TITLE
fix: windows/client-management/mdm/policy-csp-cryptography.md

### PR DESCRIPTION
**Pull request by nschonni**
- Resolve merge conflict
- Add alt text to check/x images

### CODE REVIEW

**Comment by e0i** [comment]
@manikadhiman This PR that introduces alt text to image tags is ready for review. Thanks.

File: windows/client-management/mdm/policy-csp-cryptography.md [hunk 1]
```diff
@@ -19,29 +19,20 @@ manager: dansimp
...
 
-<<<<<<< HEAD
-* [Cryptography/AllowFipsAlgorithmPolicy](#CryptographyAllowFipsAlgorithmPolicy)  
-* [Cryptography/TLSCipherSuites](#CryptographyTLSCipherSuites)
-=======
-* [Cryptography/AllowFipsAlgorithmPolicy](#cryptographyallowfipsalgorithmpolicy)  
+* [Cryptography/AllowFipsAlgorithmPolicy](#cryptographyallowfipsalgorithmpolicy)
 * [Cryptography/TLSCipherSuites](#cryptographytlsciphersuites)
->>>>>>> master
 * [Cryptography/Microsoft Surface Hub](#cryptography-policies-supported-by-microsoft-surface-hub)
```
Comment (nschonni) [inline windows/client-management/mdm/policy-csp-cryptography.md]: @ManikaDhiman you asked to delete this one, but should the the other 2 manual TOC entries above also be removed? They also get generated to the "In this article" on the right side already

Comment (ManikaDhiman) [reply windows/client-management/mdm/policy-csp-cryptography.md]: @nschonni The other two entries should be added as bullets under "Cryptography policies" head and not as manual TOC entries. They should not appear in the "In this article" list.

Comment (ManikaDhiman) [reply windows/client-management/mdm/policy-csp-cryptography.md]: Thanks for clarifying it.

Comment (nschonni) [reply windows/client-management/mdm/policy-csp-cryptography.md]: I think if you want them out of the "In this article" list, they might need to change to H3s.
If it's easier, feel free to pull/redo this internally where you can use the `review.docs.microsoft.com` site (I can't :))

Comment (ManikaDhiman) [reply windows/client-management/mdm/policy-csp-cryptography.md]: I will take care of this. You can go ahead and close this PR. Thanks for working on this!

**Pull request closed without merging**